packages/atomic/src/components/search-box-suggestions/suggestions-common.ts

```diff
@@ -4,17 +4,27 @@ import {buildCustomEvent} from '../../utils/event-utils';
 import {Bindings} from '../../utils/initialization-utils';
 import {closest} from '../../utils/utils';
 
+export interface SearchBoxDividerElement {
+  key: string;
+  content: Element | VNode;
+  onSelect?(): void;
+}
+
 export interface SearchBoxSuggestionElement {
   key: string;
-  query?: string;
-  onSelect(): void;
+  query: string;
```

Comment:
We should leave this possibly undefined, for example an instant result element won't necessarily update the query

Reply:
@olamothe @btaillon

Reply:
What's the concern with backward compatibility 🤔 ?

Reply:
I was thinking, if someone already had some suggestions without `query` in their code, it would break.. right?
but I guess since we only exposed this recently, that's OK?

Reply:
Yes, backward compatibility only kicks in when it's publicly documented/official. 

Whatever we create while designing and working on a new feature is fair game to modify and break.

Reply:
great news! I'll merge